scripts/run.py

```diff
@@ -0,0 +1,196 @@
+import requests
+import hashlib
```

Comment:
```suggestion
```

Removing the unused `hashlib` will make LGTM happy so that it will stop warning about this.